### DISCUSSION TITLE
Error 0x80070490

### DISCUSSION THREAD

**Discussion by Draxo659** [SafeExamBrowser/seb-win-refactoring#332] [Q&A] [answered]
Hello developers,
I am having a problem installing Seb application on my PC windows. The setup starts but doesn't proceeds any further and gets an error 0x80070490. I am having the same problem in my other laptop too. I have very much storage too and i have downloaded the .Net latest version too. It is really important for me so i would appreciate if this problem solution is given to me as soon as possible.

[seb error.txt](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7881778/seb.error.txt)

**Comment by dbuechel** [comment, marked as answer]
You're trying to install an older version of SEB, namely SEB 3.3.0.354. Please try the latest release version 3.3.1.

**Reply by Draxo659** [reply]
Thank u the application has been installed i am getting one more issue.
Can u check the screenshot and tell me the solution
![Screenshot (3)](https://user-images.githubusercontent.com/97889163/149942200-81417c3d-b8e9-4813-91e6-5805c02843b1.png)

**Reply by dbuechel** [reply]
Could you please provide the log files listed in the error message?

**Reply by Draxo659** [reply]
I cant find these files in my computer i tried to go through every directory but i couldn't find AppData named folder.
And when i open the Seb application they say it is not configured is that the problem how do i configure it.

**Reply by dbuechel** [reply]
Try to copy and paste the following address in the address bar of Windows explorer: `%LocalAppData%\SafeExamBrowser\Logs`

**Reply by Draxo659** [reply]
Here,
[2022-01-18_16h56m48s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7890702/2022-01-18_16h56m48s_Runtime.log)
[2022-01-18_16h56m48s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7890703/2022-01-18_16h56m48s_Service.log)

**Reply by dbuechel** [reply]
The configuration you're trying to use requires the SEB service to be running. However, according to your logs an error occurred with the service component when attempting to start a new session:

Runtime Log:
```
2022-01-18 16:57:02.191 [07] - INFO: Starting new service session...
2022-01-18 16:57:02.195 [07] - DEBUG: [ServiceProxy] Sending message 'SessionStartMessage'...
2022-01-18 16:57:02.300 [07] - DEBUG: [ServiceProxy] Received response 'SimpleResponse -> Acknowledged' for message 'SessionStartMessage'.
2022-01-18 16:57:02.303 [07] - DEBUG: [ServiceProxy] Service acknowledged session start.
2022-01-18 16:57:32.382 [07] - ERROR: Failed to start new service session within 30 seconds!
```
Service Log:
```
2022-01-18 16:57:02.565 [09] - ERROR: [EaseOfAccessConfiguration] Failed to set registry item 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\Utilman.exe\Debugger' => 'SebDummy.exe'!

   Exception Message: Attempted to perform an unauthorized operation.
   Exception Type: System.UnauthorizedAccessException

   at Microsoft.Win32.RegistryKey.Win32Error(Int32 errorCode, String str)
   at Microsoft.Win32.RegistryKey.SetValue(String name, Object value, RegistryValueKind valueKind)
   at Microsoft.Win32.Registry.SetValue(String keyName, String valueName, Object value, RegistryValueKind valueKind)
   at SafeExamBrowser.Lockdown.FeatureConfigurations.RegistryConfigurations.RegistryConfiguration.TrySet(RegistryDataItem item) in C:\Users\appveyor\projects\seb-win-refactoring\SafeExamBrowser.Lockdown\FeatureConfigurations\RegistryConfigurations\RegistryConfiguration.cs:line 226

2022-01-18 16:57:02.581 [09] - WARNING: [EaseOfAccessConfiguration] Failed to disable feature!
2022-01-18 16:57:02.595 [09] - ERROR: Failed to configure EaseOfAccessConfiguration (5cb10bd7-7f86-414f-a053-bdf48b8b26e9)!
2022-01-18 16:57:02.599 [09] - ERROR: Lockdown was not successful!
```

So if you have control over the configuration, attempt to bypass the SEB service under "Security" in the configuration tool unless you really need it and know what you're doing. In that case, you'd need to figure out why the service cant set the registry entry specified in the error log above.

**Reply by Draxo659** [reply]
Ok i will see in the security tab what can i do.

**Reply by Draxo659** [reply]
i did changes in the security thing and here is the screenshot nut the error is still the same
![Screenshot_1](https://user-images.githubusercontent.com/97889163/150518508-3b355086-191f-4152-9515-2cb595c8ceea.png)
e

**Reply by dbuechel** [reply]
You'd need to activate "Ignore SEB Service" under "SEB Service (Win)", it still is deactivated in the image above.

**Reply by Draxo659** [reply]
I deactivated it myself to see if it works or not otherwise the ignore seb was selected already can you tell me which other option I should change

**Reply by dbuechel** [reply]
> I deactivated it myself to see if it works or not otherwise the ignore seb was selected already

In that case you're most likely not starting the configuration you're actually editing. You did make sure that you're correctly editing, saving and starting the same configuration? If the ignore flag is set, SEB won't attempt to start a session with the service and the above issue should not arise.

**Reply by Draxo659** [reply]
I have trued every thing now enabled and disabled also deleted and downloaded again but it shows same thing pls can u help me with this error i need to give tolc test to a university so it's urgent.When ever i try to start the practice test they show me the password and then I copy it and automatically seb starts they ask the password and then it loads half then the error shows up.

**Reply by dbuechel** [reply]
What you can attempt is to set the following registry key `HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\Utilman.exe\Debugger` manually to `SebDummy.exe` in the registry editor before starting SEB and then restore it back to its original value after terminating SEB.

**Reply by Draxo659** [reply]
i copy and pasted the above registry key but nothing showed up can u send me in this format 
%LocalAppData%\SafeExamBrowser\Logs

**Reply by dbuechel** [reply]
You need to use the Registry Editor which is part of Windows, open the start menu and type "regedit", then start the application and navigate to the key specified above.

**Reply by Draxo659** [reply]
Ok I checked everything but i cannot find the debugger option under utilman.exe ![image](https://user-images.githubusercontent.com/97889163/151911676-e14344e9-c96f-4f37-a498-5226abd648d6.png)

**Reply by dbuechel** [reply]
You'd need to add it then, right-click and select "New" -> "String Value", name it "Debugger" and set its value to "SebDummy.exe".

**Reply by Draxo659** [reply]
I created a new string value and kept its value to seb dummey.exe but I cant name it debugger it shows an error that it cannot rename it
btw here is a screenshot of what I have done what should I do next will seb work now
![SEBDUMMY](https://user-images.githubusercontent.com/97889163/152287370-23919bf2-b77d-403e-be76-9b5143de1003.png)

**Reply by dbuechel** [reply]
> I cant name it debugger it shows an error that it cannot rename it

Did you start the registry editor with administrator privileges?

**Reply by Draxo659** [reply]
Ok so i did and still can't change the name but i added the value data so that then what should i do next will the test run now on seb.

**Reply by dbuechel** [reply]
Yes, if you managed to create the new value as specified above you can try to run SEB again and see if it fixes the problem. If not, I'd recommend that you make sure everything is working properly on your system, i.e. update to the latest Windows version and potentially check the integrity of your Windows installation.

**Reply by Draxo659** [reply]
Still the same issue 
Seb failed to start a new session! Pls consult the log files.
I updated my windows to latest version already so that there should be no possible error but still nothing happens. My windows 10 is original too.

**Reply by dbuechel** [reply]
Okay, could you then please post the most recent log files from today? Maybe there is now another issue preventing SEB from starting a new session.

**Reply by Draxo659** [reply]
OK here they are,
[2022-02-09_17h11m30s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/8057651/2022-02-09_17h11m30s_Runtime.log)
[2022-02-09_17h11m30s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/8057652/2022-02-09_17h11m30s_Service.log)

**Reply by dbuechel** [reply]
Yeah, it's still the same error. I'm afraid I can't really help you further here, there appears to be a configuration issue or some other malfunction on your system which you will need to solve first.

**Reply by Draxo659** [reply]
Oh ok but still see if u can do anything i will check from my side it is important for me as i have tolc test so pls try.

**Reply by dbuechel** [reply]
There really isn't anything more I can do from afar. I suggest you contact the technical support of your institution or bring your PC to a tech shop so that someone with professional support knowledge can have a look at the issue.

**Reply by Draxo659** [reply]
Ok i will do. And i took a new laptop and I tried on that one too but here it shows permitted or prohibited process are running and at the end it's written chrome so what should i do.
 ![image](https://user-images.githubusercontent.com/97889163/153988373-6863fb13-51bc-40c3-ac54-243172bbf299.jpeg)

**Reply by Draxo659** [reply]
And if i press yes it says seb is unable to terminate the process do it manually

**Reply by dbuechel** [reply]
> at the end it's written chrome so what should i do.

Theoretically, SEB should be able to automatically terminate Chrome if you click "OK", but if that fails for whatever reason, then you can simply terminate Chrome yourself before you start SEB.

**Reply by Draxo659** [reply]
Hi and thanks for helping me with your time. I just did factory reset for my laptop and reinstalled the windows in my laptop and updated the windows to the latest version and then downloaded All the other item needed. And now seb works. So i guess for this issue the solution would be updating the windows to latest or reinstalling windows and updating drivers. Now the application works nicely. Thanks again for your support and time dbuechel.

**Reply by dbuechel** [reply]
Excellent, glad to know that. So it thus appears to be the case that there was something corrupted with the registry on your system and yes, unfortunately the only way out of such a situation is indeed to reset resp. reinstall Windows (at least for average users).
